.travis.yml

```diff
@@ -1,21 +1,38 @@
 language: c
-compiler:
-  - gcc
-# Change this to your needs
+compiler: gcc
+addons:
+    apt:
+        packages:
+            - python3
+            - autoconf
+            - automake
+            - autopoint
+            - libtool
+            - gettext
+            - libidn11
+            - libidn11-dev
+            - libidn2-0
+            - libidn2-0-dev
+            - libicu48
+            - libicu-dev
+            - libunistring0
+            - libunistring-dev
+
 script:
-  - DIR=`pwd`
-  - git clone https://github.com/rockdaboot/libpsl
-  - cd libpsl
-  - echo "EXTRA_DIST =" >gtk-doc.make
-  - echo "CLEANFILES =" >>gtk-doc.make
-  - autoreconf --install --force --symlink
-  - OPTIONS="--with-psl-file=$DIR/public_suffix_list.dat --with-psl-testfile=$DIR/tests/test_psl.txt"
+    - DIR=`pwd`
+    - cd linter
+    - ./pslint_selftest.sh
+    - ./pslint.py ../public_suffix_list.dat
+    - cd $DIR
+    - git clone --depth=1 https://github.com/rockdaboot/libpsl
+    - cd libpsl
+    - echo "EXTRA_DIST =" >gtk-doc.make
+    - echo "CLEANFILES =" >>gtk-doc.make
+    - autoreconf --install --force --symlink
+    - OPTIONS="--with-psl-file=$DIR/public_suffix_list.dat --with-psl-testfile=$DIR/tests/test_psl.txt"
```

Comment:
I see you are still referring to the test_psl.txt file. I assume this line will have to be changed once the following change is completed:

> Libpsl is prepared to take the new tests.txt format in it's test suite. I will prepare a new release as soon as you pull the PR into master.

Am I correct?